### ISSUE TITLE
[CAIA a11y Support]: Transition Assistance Needed with Angela's Departure

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#73648]
# a11y Departure Coverage/Transition Assistance
_Angela Fowler's last day was January 12, 2024. @SarahKay8 and @sara-amanda met with Angela, prior to her departure to review any tasks that she may need our assistance on, prior to AdHoc hiring her replacement. Those tasks can be found in the task lists below. This ticket also being created to track those tasks with the CAIA Team._ 

cc: @coforma-terry @artsymartha68 

## CAIA's Transition Support for Angela

```[tasklist]
### Cartography Team AF Transition Coverage Tasks
- [x] @LynneJohnson224 (delivery manager) for the **Cartography Team**, Angela would like CAIA to reach out to them. The most-recent update is in the [DSVA MHV Cartography Team Slack](https://dsva.slack.com/archives/C0581MN69TJ/p1705087726624439) and it appears that `AF reviewed the most recent prototypes as of 1/12/2024`. _Jamie will contact team_
- [x] **Secondary Nav Research** - they are going to be studying and building [CodePen prototypes](https://codepen.io/team/va-cie/) for that, studying the secondary nav for MHV. There are a few design concepts.@sterkenburgsara [is the contact and has a prototype ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66855). There is a note in the ticket that says `a11y review` - `Timing unknown`. _Jamie will contact team_
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73798
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73999
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73996
- [x] **Capacity Calendar**: Add pilot session to a11y capacity calendar (@Sara-amanda or @coforma-jamie )
```

```[tasklist]
### MHV - Medical Records AF Transition Coverage Tasks
- [x] **Team Support** Continue to work with @BobbyBaileyRB  on MHV-related a11y needs
- [x] **Slack Support**: MHV a11y Channel `check often`
- [x] **Research**: Medical Records - `postponed per Angela`. Later in January? Check with @BobbyBaileyRB and the Medical Records Team on research timing and a11y support needs.
- [x] #77076
```

```[tasklist]
### Other AF Transition Coverage Tasks
- [x] Any other support needed `TBD`
- [x] [Video](https://youtu.be/uRvFXDiBcuM?si=p7F-4FvUUvplWDWo) and [Summary](https://docs.google.com/document/d/163IYEWUFZao08QkuPa6b6ydvyesHd50FziMEQiJq5Ww/edit?usp=sharing): Workshop - How Screen Readers Work (General) - Summary by Jamie; Video by Angela
- [x] [Video](https://youtu.be/e9XOUVoUhoU) and [Summary](https://docs.google.com/document/d/1weaSYUa5IJZNCap-4-BFwBXuSzBLIZz4gdA9xKsKHWQ/edit): Screen Reader Research Presentation - Summary by Jamie; Video by Angela
```

**Comment by coforma-jamie** [comment]
Contacted the team via Slack (see [Slack thread](https://dsva.slack.com/archives/C03Q2UQL1AS/p1706898719770819)).

**Comment by sara-amanda** [comment]
## Screen Reader Support Videos

### Workshop - How Screen Readers Work (General)
- Summary by @coforma-jamie 
- Video by Angela 
- **Links**: [Video of Workshop](https://youtu.be/uRvFXDiBcuM?si=p7F-4FvUUvplWDWo) and [Summary of Workshop](https://docs.google.com/document/d/163IYEWUFZao08QkuPa6b6ydvyesHd50FziMEQiJq5Ww/edit?usp=sharing)

### Screen Reader Research Presentation
- Summary by @coforma-jamie 
- Video by Angela 
- **Links**: [Video of Presentation](https://youtu.be/e9XOUVoUhoU) and [Summary of Presentation](https://docs.google.com/document/d/1weaSYUa5IJZNCap-4-BFwBXuSzBLIZz4gdA9xKsKHWQ/edit)
